### PR TITLE
all: Update Cloud provider kustomization

### PR DESCRIPTION
**Pull request by stevenhorsman**
- Update the kustomization files for the cloud providers now we have a single binary, so we point to the latest versions

Fixes: #643

### CODE REVIEW

**Comment by stevenhorsman** [comment]
@bpradipt - based on our quick slack conversation earlier I've created my suggestion as this PR. I'm not sure if this is on the right lines though, so have left it as a draft PR for now. If you find that I'm way off base then please feel free to update the issue with a better proposal. Thanks!

File: install/overlays/libvirt/kustomization.yaml [hunk 1]
```diff
@@ -2,10 +2,11 @@ bases:
...
 
-images:
-- name: kustomize.this/cloud-api-adaptor/image:url
-  newName: quay.io/confidential-containers/cloud-api-adaptor-libvirt # change image if needed
-  newTag: latest
+# change image if needed
+# images:
+# - name: quay.io/confidential-containers/cloud-api-adaptor:latest
+#   newName: quay.io/confidential-containers/cloud-api-adaptor
+#   newTag: release-<timestamp>
```
Comment (bpradipt) [inline install/overlays/libvirt/kustomization.yaml:9]: nit: libvirt will not be there in release builds. So the newTag can refer to `dev-<timestamp>`

Comment (snir911) [reply install/overlays/libvirt/kustomization.yaml:9]: also i think it was switched to release-<commit-id>

File: install/yamls/caa-pod.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ spec:
             name: peer-pods-secret
         - configMapRef:
             name: peer-pods-cm
-        image: kustomize.this/cloud-api-adaptor/image:url
+        image: quay.io/confidential-containers/cloud-api-adaptor:latest
```
Comment (snir911) [inline install/yamls/caa-pod.yaml:25]: how about changing this to:
```
image: cloud-api-adaptor
```
and then uncomment it in all kustomization.yaml files to:
```
images:
- name: cloud-api-adaptor
  newName: quay.io/confidential-containers/cloud-api-adaptor
  newTag: latest
```

Comment (stevenhorsman) [reply install/yamls/caa-pod.yaml]: Thanks for the suggestion - I've pushed a change based on this suggestion. Once people are is general agreement I'll squash the commits and move this PR out of draft.

File: install/overlays/libvirt/kustomization.yaml (hunk 1 above)
Comment (bpradipt) [reply install/overlays/libvirt/kustomization.yaml]: `dev-<commitid>` as per latest changes for the dev builds which includes libvirt :-)

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @stevenhorsman

**Review by snir911 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**